src/main/java/BingoApp.java

```diff
@@ -1,6 +1,50 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class BingoApp {
 
     public static void main(String[] args) {
         System.out.println("Welcome to Bingo!");
+
+        Scanner reader = new Scanner(System.in);
+        String nextGame;
+
+        do {
+            System.out.println("Starting a new game...");
+
+            ArrayList<String> names = new ArrayList<>();
+            String line;
+
+            while (names.size() < 5) {
+                System.out.println("Please enter a player's name and press enter");
+                System.out.println("When all players have been added, press enter");
+
+                line = reader.nextLine();
+
+                if (names.contains(line)) {
+                    System.out.println("Names cannot be repeated");
+                } else if (line.isEmpty() && names.size() < 2) {
+                    System.out.println("There must be at least 2 players (max. 5)");
+                } else if (line.isEmpty()) {
+                    break;
+                } else {
+                    names.add(line);
```

Comment:
Instead of adding the names to an array we can just directly add the players to the players field in the game class.